raw_data_QA/get_hash.py

```diff
@@ -0,0 +1,9 @@
+import json, sys
+
+if __name__ == '__main__':
```

Comment:
I think this could be done in a simpler way using a couple of calls to the requests library -- and then incorporated in your check_nyt_raw_data.py script. 


this is not very clean -- can be split up over more lines, but you can get the snapshot version like this, for example:
```python
import requests
prod_snapshot_version = requests.get("https://raw.githubusercontent.com/covid-projections/covidprojections/master/src/assets/data/data_url.json").json()['data_url'].split("/")[-2]

master_hash = requests.get(f"https://data.covidactnow.org/snapshot/{prod_snapshot}/version.json").json()['covid-data-public']['hash']

```

Reply:
Yeah good point, but this gets back to my earlier point about putting all the user settings about which CSVs in the top level get_data.sh script. I can change this but prefer to keep it as is.